tests/integration/preserve_ip_config_test.py

```diff
@@ -0,0 +1,136 @@
+#
+# Copyright 2018 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+import gi                           # pylint: disable=import-error
+gi.require_version('NM', '1.0')     # NOQA: F402
+from gi.repository import NM        # pylint: disable=import-error
+from libnmstate import netapplier
+
+from .testlib import statelib
+from .testlib import cmd as libcmd
+from .testlib.statelib import INTERFACES
+
+
+_EXTRA_CONFIGS = [
+    {
+        'name': 'ipv4.dad-timeout',
+        'value': '1000'
+    },
+    {
+        'name': 'ipv6.dhcp-hostname',
+        'value': 'libnmstate.example.com'
+    }
+]
+
+# TEST-NET addresses: https://tools.ietf.org/html/rfc5737#section-3
```

Comment:
I think we do not need to reason on why a specific address is used.
If it is important for you to mention it, you can do so in the commit message.